### PR TITLE
Add competition task sets and cleanup script

### PR DESCRIPTION
**Pull request by AaronGoldsmith**
## Summary
- 25 general competition tasks
- 15 agentic tasks across 3 difficulty tiers (tool-heavy, reasoning, multi-agent)
- Agent cleanup script for pruning dead weight

All tasks use relative paths and cross-platform commands.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: scripts/competition_tasks_agentic.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
++  {
++    "task": "Create a Python CLI tool at /tmp/mobius-task-csvtool/csvtool.py that reads a CSV file, supports --filter COLUMN=VALUE, --sort COLUMN, --group-by COLUMN (with count aggregation), and --output FORMAT (table or json). Generate a test CSV with at least 100 rows of synthetic employee data (name, department, salary, hire_date). Run the tool with each flag combination and verify the output is correct. Fix any bugs you find.",
++    "category": "build-and-test",
++    "tier": 1,
++    "tools_required": [
++      "Bash"
++    ],
++    "verification": "Judge checks: (1) csvtool.py exists and is executable, (2) test CSV has 100+ rows, (3) all four flag combinations produce correct output when re-run, (4) edge cases like missing columns produce helpful errors",
++    "setup": "mkdir -p /tmp/mobius-task-csvtool"
++  },
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply scripts/competition_tasks_agentic.json]: Won't do — /tmp paths are intentional for task isolation. These create throwaway artifacts outside the repo.

File: scripts/cleanup_agents.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
++
++DB_PATH = Path("data/mobius.db")
++
++
++def get_connection(db_path: Path) -> sqlite3.Connection:
++    conn = sqlite3.connect(str(db_path))
++    conn.row_factory = sqlite3.Row
++    return conn
++
++
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply scripts/cleanup_agents.py]: Addressed — stripped git metadata, file is now valid Python.

File: scripts/cleanup_agents.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
++    )
++    conn.commit()
++    return cursor.rowcount
++
++
++def clear_all_champion_flags(conn: sqlite3.Connection) -> int:
++    """Clear is_champion on all agents."""
++    cursor = conn.execute("UPDATE agents SET is_champion = 0 WHERE is_champion = 1")
++    conn.commit()
++    return cursor.rowcount
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply scripts/cleanup_agents.py]: Addressed — added elect_champions() that re-elects highest-Elo agent per specialization after clearing flags.

File: scripts/competition_tasks.json [hunk 4]
```diff
@@ -0,0 +1,125 @@
+commit fe66e75cc7f79b4ed77b2c8490f4e19862924bad
+Author: Aaron Goldsmith <aargoldsmith@gmail.com>
+Date:   Sat Mar 21 09:13:05 2026 -0700
+
+    Add agentic competition tasks, agent definitions, and skills
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply scripts/competition_tasks.json]: Addressed — stripped git metadata, file is now valid JSON with 25 task objects.

File: scripts/competition_tasks_agentic.json [hunk 5]
```diff
@@ -0,0 +1,180 @@
+commit fe66e75cc7f79b4ed77b2c8490f4e19862924bad
+Author: Aaron Goldsmith <aargoldsmith@gmail.com>
+Date:   Sat Mar 21 09:13:05 2026 -0700
+
+    Add agentic competition tasks, agent definitions, and skills
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply scripts/competition_tasks_agentic.json]: Addressed — stripped git metadata, file is now valid JSON with 16 agentic task objects.

**Comment by AaronGoldsmith** [comment]
Closing — cleanup script is a one-off utility, task JSONs can be regenerated via the competition-tasks agent.
